docker/docker-compose.prod.yml

```diff
@@ -0,0 +1,18 @@
+# Compose file for use on production servers.
```

Comment:
Sorry, that was cynical. :) I think it's okay to use compose, but I wonder if it's useful to have things specific to our production setup (port 62080, internal consul host IPs, ...) in this repo rather than docker-server-configs.